api/proto/config/devmodel.proto

```diff
@@ -56,3 +86,38 @@ message SystemAdapter {
 	// and used for app instances
 	string logicalName = 6;
 }
+
+enum PhyIoType {
+        PhyIoNoop = 0;
+        PhyIoEth = 1;
+        PhyIoUSB = 2;
+        PhyIoCOM = 3;
+        PhyIoOther = 255;
+}
+
+// How does EVE should use them, for what purpose it is for
+enum PhyIoMemberUsage {
+	PhyIoUsageNone      = 0;
+	PhyIoUsageMgmt      = 1;
+	PhyIoUsageShared    = 2;
+	PhyIoUsageDedicated = 3;
+}
+
+// PhysicalIO: 
+//    Absolute low level description of physical buses and ports that are 
+//    available on given platfrom. 
+//    Collection of these IOs, connstitue what we would call as hardware 
+//    model. Each physical IO is manageable and visible to EVE software, and
+//    it can be further configured to either provide IP connectivity or 
+//    directly be given to workloads
+message PhysicalIO {
```

Comment:
I can't review this since I don't know what any of the fields mean. Needs comments.
For instance, is the phylabel the name by which a SystemAdapter would refer to this? Or is that logicalLabel?

Nit: why start with tag 100?